tox.ini

```diff
@@ -0,0 +1,9 @@
+[tox]
+skip_missing_interpreters = True
+skipsdist = True
+envlist = py{27,34}
+
+[testenv]
+passenv = PYTHONPATH
```

Comment:
Since we don't have `setup.py`, can you add an empty `test/__init__.py` so this line is not necessary.

This way, the command becomes `py.test test/` so it doesn't accidentally run all the tests.